### DISCUSSION TITLE
All extensions supported from SD card? (Insta360 .insv files)

### DISCUSSION THREAD

**Discussion by Frankieeeeeeeeeeeee** [outdoorbits/little-backup-box#224] [Q&A] [answered]
Hi guys,

I am thinking about purchasing the Pi and running the LBB as fiddling around with smartphone and draining its battery is not perfect.
I use an Insta360 x3 camera a lot, it produces .insv (and few others) file extensions. Maybe I am blind but I did not find mention about what file extensions are supported. Can you please confirm that by default the LBB backups all the files from the SD card - so it is no problem to use with the Insta360 cameras?
I found this topic, but it was related to connection to cameras, not SD card: https://github.com/outdoorbits/little-backup-box/discussions/153

Thanks a lot!

**Comment by outdoorbits** [comment, marked as answer]
I can confirm that. Backup via cardreader will backup all files. Actually you won't be able to manage this photos in the image viewer. But if there is a need, I can check and maybe add that.

**Reply by Frankieeeeeeeeeeeee** [reply]
thanks a lot for a prompt answer! No need for image viewer, I think for 360degree footage it would be quite tricky and questionably usable. So I am going to order a Pi and start exploring this whole new world :).
